src/cljs/rems/actions/request_comment.cljs

```diff
@@ -63,10 +63,12 @@
   [{:keys [selected-commenters potential-commenters comment on-set-comment on-add-commenter on-remove-commenter on-send]}]
   [action-form-view action-form-id
    (text :t.actions/request-comment)
-   [[button-wrapper {:id "request-comment"
-                     :text (text :t.actions/request-comment)
-                     :class "btn-primary"
-                     :on-click on-send}]]
+   [[button-wrapper (merge {:id "request-comment"
```

Comment:
Wouldn't it be simpler if you did not do the merge but just set `:disabled` always?